### DISCUSSION TITLE
Where to enter wifi creds?

### DISCUSSION THREAD

**Discussion by jgdf42** [gtxaspec/wz_mini_hacks#547] [Q&A]
I was looking at wz_mini.conf, but I don't see a place to enter my wifi ssid and passphrase.

**Comment by maxfield-allison** [comment]
you do the same connect with the app that you normally would

**Comment by bjs-pdx** [comment]
> I was looking at wz_mini.conf, but I don't see a place to enter my wifi ssid and passphrase.

As @maxfield-allison said, you still need the Wyze app and you need to add the camera there first.  All the network configuration takes place in the app.
